Rewrite the onContentReady description (#2620)
DevExpress/devextreme-documentation: api-reference/10 UI Components/Widget/1 Configuration/onContentReady.md

Model data. Available only when using Knockout.

---

> The **contentReady** event fires in the following cases:     

Reviewer: ```suggestion
The **onContentReady** function is executed in the following cases:     
```

> - The component is rendered for the first time

Reviewer: ```suggestion
- The component is rendered for the first time.
```

> - The data is fully loaded

Reviewer: ```suggestion
- Data is fully loaded.
```

> - Each time the content is changed

Reviewer: ```suggestion
- Each time the content is changed.
```

> - The [repaint()](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Methods/#repaint) method

Reviewer: ```suggestion
- The [repaint()]({basewidgetpath}/Methods/#repaint) method is called.
```
